### DISCUSSION TITLE
Fix scroll bar on `edit`/`preview` code screen

### DISCUSSION THREAD

**Discussion by piradata** [community/community#54349] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Select Feature Area

- [X] Issues
- [ ] Projects

### Fix scroll bar on `edit`/`preview` code screen

Look at the scroll bar on the `edit` tab

![image](https://user-images.githubusercontent.com/16529503/232682872-e31d2a41-6fde-4f0d-8d10-4b09ee46d510.png)

Now look at the scroll bar on the `preview` tab

![image](https://user-images.githubusercontent.com/16529503/232682844-ebd202e3-8ddb-4d3f-91ce-3011036b2260.png)

Should not both be inside the container?

When I scroll down in the `preview` tab I can't go back to `edit` unless I scroll all the way to the top before. And that does not happens on the `edit` tab.

![image](https://user-images.githubusercontent.com/16529503/232683427-390b4730-bcb9-4e39-adb9-c888b0a3a04f.png)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by tuves** [comment]
Hi @piradata,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post. I've moved your other post to the correct category and added a bug label.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
